backend/corpora/dataset_processing/process.py

```diff
@@ -218,10 +216,52 @@ def make_cxg(local_filename):
     return cxg_dir
 
 
-def main():
+def copy_cxg_files_to_cxg_bucket(cxg_dir, bucket_prefix):
+    command = ["aws"]
+    if os.getenv("BOTO_ENDPOINT_URL"):
+        command.append(f"--endpoint-url={os.getenv('BOTO_ENDPOINT_URL')}")
+    command.extend(
+        [
+            "s3",
+            "cp",
+            cxg_dir,
+            f"s3://{os.environ['CELLXGENE_BUCKET']}/{bucket_prefix}.cxg/",
+            "--recursive",
+            "--acl",
+            "bucket-owner-full-control",
+        ]
+    )
+    subprocess.run(
+        command,
+        check=True,
+    )
 
-    check_env()
 
+def convert_files_ignore_exceptions(local_filename):
+    exceptions = []
+    try:
+        cxg_dir = make_cxg(local_filename)
+    except Exception as e:
+        cxg_dir = None
+        print(f"Issue creating cxg: {e}")
```

Comment:
So this swallows the traceback doesn't it? That makes it a bit harder to debug.

Reply:
discussed in slack -- this will be addressed by a pr by @Bento007 which switches from print to logger.exception

Reply:
👍